HW02_B_Mimic_starter.Rmd

```diff
@@ -52,7 +60,11 @@ data("iris")
 
 Using the iris dataset, make this graph:
 ```{r graph 2 code, echo=FALSE}
-
+iris$Species_order <- factor(iris$Species, levels = c("versicolor", "setosa", "virginica"))

```

Comment:
Very nice! So you essentially define a new variable inside the iris dataset to change the order of the species, and then use that variable for plotting? I couldn't find out how to change the order of the panels, but it seems like this works!

Reply:
You are right. To be honest, I also got inspiration from another classmate's HW.